docker-client/openapi.json

```diff
@@ -0,0 +1,118 @@
+{
```

Comment:
I don't see any code to generate this file, did you just manually copy/paste it? If we aren't autogenerating this file probably better not to have it at all and run `poetry run openapi-python-client generate --url http://localhost:3000/api/internal/openapi.json` or whatever.

Reply:
Nvm I see where you generate this file now.